src/codeu/chat/util/Encryptor.java

```diff
@@ -0,0 +1,141 @@
+/**
+ * Encryptor.java
+ * Provides various encryption-related methods.
+ */
+
+package codeu.chat.util;
+
+import javax.crypto.*;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.security.*;
+
+public class Encryptor {
+    private static final String SYMMETRIC_ALGORITHM = "AES";
+    private static final String ASYMMETRIC_ALGORITHM = "RSA";
+
+    public static SecretKey makeSymmetricKey() {
+        KeyGenerator keyMaker;
+
+        try {
+            keyMaker = KeyGenerator.getInstance(SYMMETRIC_ALGORITHM);
+        } catch(NoSuchAlgorithmException nsae) {return null;}
```

Comment:
Catching the exception and returning null on errors is not a good idea. Log the full stack trace and exit the application instead, since it is not a recoverable error.